application/actions/post_test.go

```diff
@@ -112,6 +115,10 @@ func (as *ActionSuite) Test_PostQuery() {
 	as.Equal(f.Posts[0].Title, resp.Post.Title)
 	as.Equal(f.Posts[0].Description.String, resp.Post.Description)
 
+	wantDate := f.Posts[0].NeededBefore.Time
+	wantDate = time.Date(wantDate.Year(), wantDate.Month(), wantDate.Day(), 0, 0, 0, 0, time.UTC)
```

Comment:
Is this right? Looks the the first assignment is not used.

Reply:
It's dropping off the hours-minutes... from the fixture time.

Reply:
Oh, I see it now. Sorry, read too quickly.